app/assets/stylesheets/application.scss

```diff
@@ -61,30 +70,24 @@ $indianred: #e45e56;
   text-align: center;
 }
 
-// Footer
-
-.page-wrap {
-	margin-bottom: -142px;
-  min-height: 100%;
-}
-
-.page-wrap:after {
-  content: "";
-  display: block;
-}
+$footer-height: 100px;
 
-.site-footer,
-.page-wrap:after {
-  height: 142px;
+.wrapper {
+  min-height: calc(100% - #{$footer-height});
+	background-image: image_url("turks.jpg");
+	background-size: cover;
+	background-repeat: repeat-y;
+	background-position: center top;
+	background-position: 0px 45px;
```

Comment:
Property `background-position` already defined on line 80<br>Line should be indented with spaces, not tabs<br>`0px` should be written without units as `0`